prac_06/guitars.py

```diff
@@ -0,0 +1,20 @@
+from prac_06.guitar import Guitar
+guitars = [Guitar("Gibson L-5 CES", 1922, 16035.40), Guitar("Line 6 JTV-59", 2010, 1512.9)]
+print("My guitars!")
+while True:
```

Comment:
Can use "while name != "": " as your while argument so save you using an if statement to break the loop.